collectors/macos.plugin/multi_metadata.yaml

```diff
@@ -4,91 +4,64 @@ modules:
       plugin_name: macos.plugin
       module_name: mach_smi
       monitored_instance:
-        name: macos mach_smi
-        link: ''
-        categories: []
-        icon_filename: ''
+        name: MacOS
+        link: "https://www.apple.com/macos"
```

Comment:
Does this get used anywhere on the UI? Couldn't find it being used.

Reply:
no it does not, I could not find a good way to use it at the moment.

We don't artificially construct a sentence having the link, nor do we specially tag a string to be replaced with the hyperlink, so for the moment it is not used.

Reply:
it is nice to have at the moment so on future iterations we can inject it in.
But second thinking, why funnel users out from app and onto an external site about the integration, this is a double edged sword, so we have to be careful how we implement it.